lib/control-app.js

```diff
@@ -25,10 +41,15 @@ module.exports = function(config) {
 
     var build;
     // default builder is browserify which we provide
-    config.builder = config.builder || '../lib/builder-browserify';
+    config.builder = config.builder || defaultBuilder;
```

Comment:
@vvo would be better to use the same API for both builders. Maybe I wasn't clear before but I am ok with making a breaking API change for the builder API since it leads to better code architecture for the builder.